src/net.rs

```diff
@@ -59,6 +59,7 @@ use byteorder::{ByteOrder, NativeEndian, NetworkEndian};
 use std::io::{BufRead, BufReader, Read};
 use std::net::{Ipv4Addr, Ipv6Addr, SocketAddr, SocketAddrV4, SocketAddrV6};
 use std::{path::PathBuf, str::FromStr};
+use libc::pid_t;
```

Comment:
I would to not add the `libc` dep, just for the `pid_t` type.  Can we just use an `i32` (which is what we also use in the `Process` module) ?